ArchUnitNET/Domain/PropertyMember.cs

```diff
@@ -38,6 +39,8 @@ public class PropertyMember : IMember, ITypeInstance<IType>
 
         [CanBeNull] public MethodMember Setter { get; internal set; }
 
+        public bool IsInitSetter { get; }
```

Comment:
I'd like to bring up again the suggestion of @fholger from [a previous PR](https://github.com/TNG/ArchUnitNET/pull/170#pullrequestreview-1083758005) to unify the flags `IsInitSetter` and `IsReadOnly` into a single enum property with three states. As a discussion starter, I would suggest
```cs
enum AccessRight { ReadOnly, ReadInit, ReadWrite }
```

Reply:
I like that idea, but I'm thinking it would be better to separate write from read accessors and create an enum for like this:
```c#
enum WriteAccessor { ReadOnly, Init, Set }
```
If it had to be for both read and write accessors, it would have to look something like that:
```c#
enum Accessor { ReadOnly, ReadInit, ReadSet, InitOnly, SetOnly }
```
What do you think about it?

Reply:
The write accessor enum idea sounds good to me 👍